### PR TITLE
docs: improve getting started and add personal space transfer (#3210)

### PR DESCRIPTION
**Pull request by Kroustille**
# Context
> Briefly describe the purpose and context of this PR. Include relevant screenshots or screen recordings for the product team.
>
> ℹ️ **Feature environment** — A dedicated preview environment (`https://dev-pr-{number}.hub.staging.filigran.io`) is automatically deployed when CI passes. Add the `skip-feature-env` label to opt out.

Architecture and technical requirements were not up to date with current project.
Personal space transfer feature was missing from documentation.

Nothing to test, documentation only.

## Related issues

- Related to #3210

# Checklist

## Quality

- [ ] I consider this work complete and ready for review
- [ ] I tested all features and edge cases
- [ ] I refactored code where necessary to improve quality
- [ ] I made sure my code meets development guidelines
- [ ] I performed a self-review of my code
- [ ] I added the `skip-feature-env` label if a feature environment is not needed for this PR

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [ ] Local manual tests

# Additional information
> Any additional context, dependencies, or concerns reviewers should know about.

### CODE REVIEW

**Review by elisavaldegas — APPROVED** [review]
functional part looks good thank you :)

File: docs/docs/getting-started.md [hunk 1]
```diff
@@ -89,70 +99,71 @@ This starts:
...
 
-### Frontend development
+### Frontend (`apps/frontend`)
 
 - **Development mode**: `yarn dev`
+- **Type check**: `yarn check-ts`
+- **Lint**: `yarn lint`
 - **Run tests**: `yarn test`
-- **Generate GraphQL types**: `yarn relay`
+- **Generate Relay/GraphQL artifacts**: `yarn relay` (**required** after any GraphQL schema change)
```
Comment (hervyt) [inline docs/docs/getting-started.md]: **suggestion** Maybe we should indicate Relay is now deprecated and should use React Query

**Review by hervyt — APPROVED** [review]
